.gitignore

```diff
@@ -11,3 +11,6 @@ packages
 
 # Include when developing application packages.
 pubspec.lock
+
+.pub
```

Comment:
If you would't mind, could you make these `.pub/` and `.idea/`? Makes it a bit clearer that these are directories that are being ignored.